data/mainnet/pools.js

```diff
@@ -4,6 +4,20 @@ const addresses = require('./addresses.json')
 const strat30PercentFactor = '0.7'
 
 module.exports = [
+  {
+    chain: CHAINS_ID.ETH_MAINNET,
+    id: 'UniV3_ORC_ETH',
+    tradingApyFunction: {
+      type: TRADING_APY_TYPES.UNIV3_V3,
```

Comment:
Let's use the apyvision one. It does not have any data right now, as the position was only created today, but it will have data when the vault goes live.